pkg/api/v20170701/types.go

```diff
@@ -184,6 +184,26 @@ type AgentPoolProfile struct {
 	subnet string
 }
 
+// PoolUpgradeProfile contains pool properties:
```

Comment:
does it mean each pool may have different orchestrators?

Reply:
Each pool may potentially have different orchestrator versions. This is not implemented yet.